src/qt/locale/bitcoin_ar.ts

```diff
@@ -1394,274 +1380,274 @@
-        <translation type="unfinished"></translation>
+        <translation>يوصى بفصل الإنترنت قبل طباعة المحافظ الورقية. على الرغم من إنشاء المحافظ الورقية على جهاز الكمبيوتر المحلي الخاص بك، لا يزال من الممكن أن يكون لديك برامج ضارة تنقل شاشتك إلى مكان بعيد دون قصد. يوصى أيضًا بالطباعة إلى طابعة محلية مقابل طابعة شبكة حيث يمكن مراقبة حركة مرور الشبكة. تقوم بعض الطابعات المتقدمة أيضًا بتخزين نسخ من كل مستند مطبوع. تابع بحذر فيما يتعلق بمقدار القيمة التي تخطط لتخزينها على كل عنوان.</translation>
     </message>
     <message>
         <source>Error encoding Address into QR Code.</source>
-        <translation type="unfinished"></translation>
+        <translation>خطأ في ترميز العنوان في رمز الاستجابة السريعة</translation>
     </message>
     <message>
         <source>Error encoding private key into QR Code.</source>
-        <translation type="unfinished"></translation>
+        <translation>خطأ في ترميز المفتاح الخاص في رمز الاستجابة السريعة</translation>
     </message>
     <message>
         <source>Printing Error</source>
-        <translation type="unfinished"></translation>
+        <translation>خطأ في الطباعة</translation>
     </message>
     <message>
         <source>failed to open file, is it writable?</source>
-        <translation type="unfinished"></translation>
+        <translation>فشل في فتح الملف، هل هو قابل للكتابة</translation>
     </message>
     <message>
         <source>Load Paper Wallets</source>
-        <translation type="unfinished"></translation>
+        <translation>تحميل محافظ الورق</translation>
     </message>
     <message>
         <source>The paper wallet printing process has begun.&lt;br/&gt;Please wait for the wallets to print completely and verify that everything printed correctly.&lt;br/&gt;Check for misalignments, ink bleeding, smears, or anything else that could make the private keys unreadable.&lt;br/&gt;Now, enter the number of DOGE you wish to send to each wallet:</source>
-        <translation type="unfinished"></translation>
+        <translation>بدأت عملية طباعة المحفظة الورقية.;يُرجى الانتظار حتى تتم طباعة المحافظ بالكامل والتحقق من طباعة كل شيء بشكل صحيح.;تحقق من وجود أخطاء في المحاذاة أو نزيف الحبر أو البقع أو أي شيء آخر قد يؤدي إلى المفاتيح الخاصة غير قابلة للقراءة.;الآن، أدخل رقم دوج الذي ترغب في إرساله إلى كل محفظة</translation>
     </message>
     <message>
         <source>Paper wallet %1</source>
-        <translation type="unfinished"></translation>
+        <translation>المحفظة الورقية %1</translation>
     </message>
     <message>
         <source>&lt;b&gt;%1&lt;/b&gt; to Paper Wallet &lt;span style=&apos;font-family: monospace;&apos;&gt;%2&lt;/span&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>إلى المحفظة الورقية</translation>
```

Comment:
The translation is missing `%1` and `%2`

The `%1` is an amount (ie 8.0) and the `%2` is an address.

So for example, the resulting string in this case could become, in English:

```
8.0 to Paper Wallet DDogepartyxxxxxxxxxxxxxxxxxxw1dfzr
```

Hope that gives better context to what this is for. Because the html won't necessarily work well with arabic, I'd suggest leaving them out for now.